osc_qasm.py

```diff
@@ -11,18 +11,66 @@
+        the OSC-Qasm client
+
+        usage: sys.stderr = FileLikeErrorOSC()
+        '''
+    def __init__(self):
+
+        self.older="" # stderr 'memory'
+
+    def write(self, text):
+        if text != f'\n' and text != "": # Skips end='\n'|'' argument messages
+            print(text) # Print back to console
+
+            if text == ERR_SEP and self.older != ERR_SEP and self.older != "": # There is a line like ERR_SEP both at the begining and end of a qiskit error log!
+                # Print the last entry before the ending ERR_SEP
+                client.send_message("error", "error in osc_qasm.py: \n(...) "+self.older+"switch to python console to learn more")
+
+            elif "KeyboardInterrupt" in text:
+                # When closing the program with Ctrl+c, There is a 'KeyboardInterrupt' error message.
+                client.send_message("error", "osc_qasm.py has been terminated in the Python environment.")
+
+            self.older=text # Update memory
 
 def run_circuit(qc, shots, backend_name):
 
     print("Running circuit on {}...".format(backend_name))
     client.send_message("info", "Running circuit on {}...".format(backend_name) )
+
+    flosc = FileLikeOutputOSC() # Use this only for job_monitor output
+
     if backend_name != 'qasm_simulator':
         if backend_name in ('FakeAlmaden', 'FakeArmonk', 'FakeAthens', 'FakeBelem', 'FakeBoeblingen', 'FakeBogota', 'FakeBrooklyn', 'FakeBurlington', 'FakeCambridge', 'FakeCambridgeAlternativeBasis', 'FakeCasablanca', 'FakeEssex', 'FakeGuadalupe', 'FakeJakarta', 'FakeJohannesburg', 'FakeLagos', 'FakeLima', 'FakeLondon', 'FakeManhattan', 'FakeManila', 'FakeMelbourne', 'FakeMontreal', 'FakeMumbai', 'FakeOurense', 'FakeParis', 'FakePoughkeepsie', 'FakeQuito', 'FakeRochester', 'FakeRome', 'FakeRueschlikon', 'FakeSantiago', 'FakeSingapore', 'FakeSydney', 'FakeTenerife', 'FakeTokyo', 'FakeToronto', 'FakeValencia', 'FakeVigo', 'FakeYorktown'):
             backend_name+='()'
             backend = eval(backend_name) # this is definitely a security hazard... use at your own risk!
-            # a very interesting alternative is to use: backend = globals()[backend_name]
+                # a very interesting alternative is to use: backend = globals()[backend_name]
             available_qubits = backend.configuration().n_qubits
-            requested_qubits = qc.num_qubits            
+            requested_qubits = qc.num_qubits
             if requested_qubits > available_qubits: # verify if the qubit count is compatible with the selected backend
                 client.send_message("error", "The circuit submitted is requesting {} qubits but the {} backend selected only has {} available qubits.".format(requested_qubits,backend_name[:-2],available_qubits) )
                 raise ValueError('The circuit submitted is requesting {} qubits but the {} backend selected only has {} available qubits.'.format(requested_qubits,backend_name[:-2],available_qubits))
```

Comment:
```suggestion
                print('The circuit submitted is requesting {} qubits but the {} backend selected only has {} available qubits.'.format(requested_qubits,backend_name[:-2],available_qubits))
                sys.exit()
```
There, this should do it :)

Reply:
Nice solve!

Reply:
In this way, there is a client.send _message(error) for Max, a print for command-line, and the system exits whitout duplicating the message with 'raise ValueError'